src/App.js

```diff
@@ -9,6 +9,7 @@ import {
 import Page from './layout/Page/Page'
 import Loader from './common/Loader/Loader'
 
+import { useMode } from './hooks/demoMode.hook'
```

Comment:
know this hook isn't only for `demo` mode. So, you can rename the file to `mode.hook`